Mod/MultiplayerModeCore/CMakeLists.txt

```diff
@@ -0,0 +1,84 @@
+project(MultiplayerModeCore)
+
+set(MULTIPLAYERMODECORE_BIGOBJ_FILES
+	Libs/Arise-SDK/src/SDK/Arise_functions.cpp
+)
+
+set(MULTIPLAYERMODECORE_FILES
```

Comment:
As someone who only uses CMake from time to time: Is it possible to group the files? I.e. separate the external classes from our own or keep the modules in their own filter / directory?